item.rb

```diff
@@ -1,8 +1,9 @@
 class Item
-  attr_reader :id, :genre, :author, :source, :label, :publish_date, :archieved
+  attr_accessor :genre, :author, :source, :label, :publish_date
+  attr_reader :id, :archieved
 
-  def initialize(id, publish_date, archieved)
-    @id = id
+  def initialize(publish_date, archieved)
```

Comment:
It seems like the archieved typo appeared here also.

Reply:
@teckim **I had resolved the Typos as requested by you.**